### ISSUE TITLE
psnr/ssim

### ISSUE THREAD

**Issue by XijieJiao** [csmliu/STGAN#40]
Hi, which file do you use to compute the psnr/ssim of your results?
Thank you.

**Comment by csmliu** [comment]
Kindly refer to https://github.com/csmliu/STGAN/issues/20#issuecomment-534065361

**Comment by XijieJiao** [comment]
Thank you for your reply. I notice that the code is for calculation of one attribute manipulation. The paper shows that the psnr/ssim is 31.67/0.948. What attribute or attributes are these two numbers point to? 
By the way, I tested the attribute 'Eyeglasses' and used a test set of 6000 images from your test set. The result of your trained model is 26.524815741791713 / 0.9064622972518696. I also retrained on your training set following the paper's instruction and the retrained the model got a psnr/ssim of 27.0123/0.9045. I am wondering why these two gaps happens.
Thank you.

**Comment by csmliu** [comment]
Hi Xijie,
The PSNR and SSIM are used to evaluate the reconstruction output images, since we have no groundtruth of manipulated images.
And the PSNR and SSIM on the reconstruction images are used to indirectly show the image quality.

**Comment by XijieJiao** [comment]
What do you mean by reconstruction output images? The results of editing and editing again(reversion)?

**Comment by csmliu** [comment]
No, the reconstruction output images are generated by `att_diff=0` for STGAN, and `att_target=att_source` for others
